dspace-jspui/src/main/java/org/dspace/app/webui/discovery/DiscoverUtility.java

```diff
@@ -466,10 +466,14 @@ private static void setPagination(HttpServletRequest request,
             //
             // }
 
-            if (rpp > 0)
+            if (rpp > 0 && rpp < discoveryConfiguration.getMaxRpp())
```

Comment:
this should be `if (rpp > 0 && rpp <= discoveryConfiguration.getMaxRpp())`

Reply:
Fixed.